src/components.d.ts

```diff
@@ -15,6 +15,7 @@ import { Direction, Type } from "tidily";
 import { Criteria } from "selectively";
 import { Data } from "./model/Data";
 import { Looks } from "./components/input/Looks";
+import { Notice as Notice1 } from "./model/Notice";
```

Comment:
This looks like it's imported twice.
Once here from `"./model/Notice"` and once from `"./model"` on line 11.

Reply:
the import can change if it is imported as 
`import { model } from "../something/model` vs `import { Notice } from "../somtething/model/Notice` so it means that the imports are mixed in the components it should be imported as the first one.

Reply:
Sorry missed that this was `components.d.ts`.